addons/mail/models/mail_link_preview_message.py

```diff
@@ -0,0 +1,48 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from odoo.addons.mail.tools.discuss import Store
+
+
+class LinkPreviewMessage(models.Model):
+    _name = "mail.link.preview.message"
+    _inherit = "bus.listener.mixin"
```

Comment:
Same here, ORM wants to enforce always using a list
```suggestion
    _inherit = ["bus.listener.mixin"]
```